UnitySDK/Assets/TiltBrush/Scripts/Editor/ModelImportSettings.cs

```diff
@@ -33,16 +33,19 @@ public class ModelImportSettings : AssetPostprocessor {
 
   // UVs come as four float2s so go through them and pack them back into two float4s
   static void CollapseUvs(Mesh mesh) {
-    var finalUVs = new List<List<Vector4>>();
+    var finalUVs = new List<List<Vector4>>(mesh.vertexCount);
+
     for (int iUnityChannel = 0; iUnityChannel < 2; iUnityChannel++) {
-      var sourceUVs = new List<Vector2>();
-      var targetUVs = new List<Vector4>();
+      var sourceUVs = new List<Vector2>(mesh.vertexCount);
+      var targetUVs = new List<Vector4>(mesh.vertexCount);
       int iFbxChannel = 2 * iUnityChannel;
       mesh.GetUVs(iFbxChannel, targetUVs);
       mesh.GetUVs(iFbxChannel + 1, sourceUVs);
-      if (sourceUVs.Count > 0 || targetUVs.Count > 0) {
-        for (int i = 0; i < sourceUVs.Count; i++) {
-          if (i < targetUVs.Count) {
+      int sourceUVsCount = sourceUVs.Count;
+      int targetUVsCount = targetUVs.Count;
```

Comment:
Not safe to cache this value if it's going to change (see targetUVs.Add())

Reply:
This should actually be safe because once i>targetUV's  initial size, i will always be greater than targetUV's size. targetUVs.Add will always increase targetUVs.Count to match sourceUVs previous index.

Reply:
I agree that the code as written works, but the name "targetUVsCount" misleads the reader into thinking that it has identical semantics to "targetUVs.Count"; that's the dangerous thing. But again, don't worry about updating this PR until my rewrite lands